### ISSUE TITLE
build.sh created with Windows line endings

### ISSUE THREAD

**Issue by psfblair** [fsharp-editing/Forge#54]
I'm on macOS Sierra using Forge through Ionide. Just generated a new project and the build.sh has Windows line endings.

## Expected Behavior
Running build.sh at the command line should not produce errors.

## Current Behavior
Running build.sh fails with "build.sh: line 34: syntax error: unexpected end of file"

## Possible Solution
Diffing the build.sh against another (working) project I generated just a few days ago shows that the only difference in build.sh is that the non-working one has CRLF line endings.

## Steps to Reproduce (for bugs)
1. In VS Code 1.10.1 with Ionide 2.23.9, Ionide-FAKE 1.2.3 and Ionide-Paket 1.6.3 open the directory that will be the root folder of the project.
2. In the command palette, use the "F#: New Project" command, select a classlib, the directory src, and a name for the project. 
3. Wait a bit for the build files to be generated.
4. Open build.sh in the editor. The editor mode will be CRLF. (This can also be verified using other tools.)
5. Try using the command "FAKE: Build Default"; the command will fail with a syntax error.

## Context
Can't run builds without changing the line endings of the file.

## Your Environment
* Version used: Unclear how to determine what version is in Ionide 2.23.9
* Environment name and version (F#, .Net): mono 4.4.0, F# 4.0
* Operating System and version: macOS Sierra

**Comment by Krzysztof-Cieslak** [comment]
I think we've recently updated Forge templates and fixed issue with line endings. Try running `F#: Refresh Project Templates` and check if problem is still reproducible

**Comment by psfblair** [comment]
That worked. However, two things:

1. Last line of console output is:
```
Unhandled error:
Could not find file "/Users/[user]/Documents/[path_to_project]/src/build.sh".

```
Not sure why since the file is there.

2. The last line of the output when refreshing the project templates is the `git clone` command.  As far as I can tell there's no output from git clone and also no output indicating that the command to refresh templates is finished.

2. The

**Comment by Krzysztof-Cieslak** [comment]
Should be fixed as part of #71
